frontend/public/components/utils/dropdown.jsx

```diff
@@ -474,6 +474,7 @@ class Dropdown_ extends DropdownMixin {
                       className="pf-c-form-control"
                       onClick={(e) => e.stopPropagation()}
                       data-test-id="dropdown-text-filter"
+                      data-test="dropdown-text-filter"
```

Comment:
I'd just use the existing legacy test ID for now instead of having two test IDs.